### ISSUE TITLE
Node 0.12 binaries

### ISSUE THREAD

**Issue by patriksimek** [jorgeazevedo/node-sqlserver-unofficial#7]
It would be nice to have this module updated with latest-node-compatible binaries.

**Comment by jorgeazevedo** [comment]
Thanks for getting in touch! I unfortunately do not have access to an azure account anymore. Would you be willing to compile some binaries and open a pull request?

There's a lot of forks to this project, maybe some binaries are already available?

**Comment by tysjiang** [comment]
Also would be extremely happy to see this updated for node 12 (sorry, can't help with the binaries though)

**Comment by jorgeazevedo** [comment]
@patriksimek @tysjiang 
tldr: no can do. node-sqlserver does not compile with node 0.12.

I had some time to look into this. I setup the build environment on a x86 windows 7 VM and was able to compile with node 0.10 just fine. With node 0.12 it fails. It seems there were some interface changes at the V8 engine level. Many method calls `node-sqlserver` does are simply not available. Here's an example of a compilation error

```
c:\users\jorgeazevedo\documents\node-sqlserver\src\Utility.h(30): error C2248: 'v8::HandleScope::HandleScope' : cannot access protected member declared in class 'v8::HandleScope' [c:\Users\jorgeazevedo\Documents\node-sqlserver\build\sqlserver.vcxproj]
          c:\Users\jorgeazevedo\.node-gyp\0.12.0\deps\v8\include\v8.h(816) : see declaration of 'v8::HandleScope::HandleScope'
          c:\Users\jorgeazevedo\.node-gyp\0.12.0\deps\v8\include\v8.h(800) : see declaration of 'v8::HandleScope'
```

TThe C++ code needs to be rewritten so the functionality is implemented using the 0.12 V8 API. This is beyond me.

`node-sqlserver` is mostly dead these days. There are no Microsoft resources actively maintaining or developing this project. Support for this will have to come from the community. As of yet, I saw no talk of it in the project's issues, nor any forks with work in this direction.

If someone does implement it, I'll be glad to ship it :-) Until then, I'm closing this issue.
